internal_review/housing_security/borough/income_restricted_units.csv

```diff
@@ -0,0 +1,6 @@
+borough,NYCHA_units,RAD_units
+BK,54681.0,3940.0
+BX,40866.0,2458.0
```

Comment:
I don't think it makes sense to keep this as a float, should be an integer. I don't know if OSE cares but just want to flag